### ISSUE TITLE
Line number in draft

### ISSUE THREAD

**Issue by neuronalX** [ReScience/ReScience#34]
I am currently reviewing a paper, and I think it would be more convenient to have the number of lines written on the submitted paper draft, so reviewers could refer more easily to particular expressions or sentences in the text.

**Comment by rougier** [comment]
Yes, good idea. It's only a matter of adding a package in the latex template.
